pkgs/phps.nix

```diff
@@ -43,6 +43,9 @@ let
 
               "--enable-libxml"
               "--with-libxml-dir=${prev.libxml2.dev}"
+            ]
+            ++ prev.lib.optionals (prev.lib.versionAtLeast args.version "7.3") [
+              "--with-pcre-regex=${prev.pcre2.dev}"
```

Comment:
Actually, not sure about this. Should not it be `versionOlder args.version "7.4"`? I would expect newer PHP versions to pick it up using pkg-config.

Reply:
I'm not sure either, maybe @shyim or @HelloWorld017 knows?

Reply:
Dammit...

Reply:
I'll open a new PR right now. Sorry about that.